### ISSUE TITLE
Internet Explorer 11: Object doesn't support property or method 'forEach'

### ISSUE THREAD

**Issue by schleifer-john** [tim-evans/ember-file-upload#139]
I have been troubleshooting File Uploads locally in Internet Explorer 11 and get the following error when running my project:

> SCRIPT438: Object doesn't support property or method 'forEach'

The source of the issue is this line of code: https://github.com/tim-evans/ember-file-upload/blob/4a755cd78a962e8e5a21762071cbfa6598a7d1c3/addon/components/file-upload/component.js#L182

There is a similar ticket in the Babel project: https://github.com/babel/babel/issues/6511
Basically the root cause of the issue is that IE 11 does not support the 'forEach' method on 'querySelectorAll'.

https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach#Browser_Compatibility

There are some suggestions in the Babel link of how to fix the issue.  I noticed that the offending code in this project is wrapped in a 'DEBUG' block, so it may not be critical to fix.

**Comment by tim-evans** [comment]
We could `Array.prototype.slice.call(this.element.querySelectorAll('*'))` to `toArray` this

**Comment by kumaresan-subramani** [comment]
this helps me

https://github.com/miguelcobain/ember-paper/issues/1058#issuecomment-461764542

**Comment by jelhan** [comment]
@kumaresan-subramani You should update to a recent version of ember-file-upload. It doesn't use `querySelectorAll().forEach` anymore: https://github.com/adopted-ember-addons/ember-file-upload/blob/f11521afa1649f6adcbf288a4043b108d31b673a/addon/components/file-upload/component.js#L195-L202
